pex/pex_info.py

```diff
@@ -53,13 +50,15 @@ class PexInfo(object):
   @classmethod
   def make_build_properties(cls, interpreter=None):
     from .interpreter import PythonInterpreter
-    from pkg_resources import get_platform
+    from .platforms import Platform
```

Comment:
This relative import initially made me uncomfortable because I know they can be problematic for things like refactoring, and the Py2 syntax breaks on Py3. But I reviewed the book Python Cookbook and they say it's totally valid + you're copying the precedent, so I think it's fine.

You're also using the Py3 syntax, which resolves the biggest concern I would have.